lib/rspec/core/formatters/json_formatter.rb

```diff
@@ -42,6 +42,10 @@ def stop(notification)
                   :backtrace => e.backtrace,
                 }
               end
+              pm = example.execution_result.pending_message
+              if pm
+                hash[:pending_message] = pm
+              end
```

Comment:
IMO, it would be better to _always_ assign a `:pending_message` key.  I (and many other experienced rubyists) have a preference for using `Hash#fetch` rather than `Hash#[]` for accessing keys in a hash created by someone else...that way, I get a clear `KeyError` if the key is renamed upstream or if I typo it.  If I use `Hash#[]` and get `nil`, I have no idea if I mispelled the key, it got renamed or just doesn't have a value.  It would be like having an object that only responds to a particular message if it has a value to return for it.